### DISCUSSION TITLE
How to remove certain features from the dev kit?

### DISCUSSION THREAD

**Discussion by soulfire801** [FreeTubeApp/FreeTube#6424] [Q&A] [answered]
Hello, I see freetube is open source so I’m trying to develop a personalized installation of free tube for my special needs child. The only difference being this is one is without the search bar or comment section. I want him to only access the video subscriptions I provide for him. However I’m kinda a noob when it comes to Java app development. So my question is for someone who is experienced in this field: what is the code I have to remove in the development kit in order to achieve this? Thanks in advance!

**Comment by kommunarr** [comment, marked as answer]
Hi @soulfire801, we have a `Hide Comments` option under Distraction Free Settings and a `Hide Search Bar` option under Parental Control Settings, so you should be good for the use case you've described! You can then set a password for modifying these under Password Settings. If you do want to recommend and/or develop any new parental control features, do note that FreeTube is made in JavaScript, not Java. [Here's the Development section of our documentation for help on getting set up to contribute.](https://docs.freetubeapp.io/development/)
